engine/src/test/scala/cromwell/engine/WorkflowDescriptorSpec.scala

```diff
@@ -171,7 +171,9 @@ class WorkflowDescriptorSpec extends FlatSpec with Matchers with ScalaFutures {
           1,
           Map.empty,
           None,
-          None)
+          None,
+          false,
+          false)
         call -> Seq(metadata)
     }).toMap
 
```

Comment:
Actually there are some nice tests in `RetryableCallsSpec` and `WorkflowManagerActorSpec`.